src/AppBundle/Committee/CommitteeExportTransformer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace AppBundle\Committee;
+
+use AppBundle\Entity\Adherent;
+
+class CommitteeExportTransformer
+{
+    /**
+     * @param Adherent[] $adherents
+     *
+     * @return string
+     *
+     * @throws \BadMethodCallException
+     */
+    public static function getAdherentsAsCsv(array $adherents)
+    {
+        $handle = fopen('php://memory', 'r+');
+        fputcsv($handle, ['Prénom', 'Nom', 'Age', 'Ville', 'Pays', 'Adresse email']);
+
+        foreach ($adherents as $adherent) {
+            if (!$adherent instanceof Adherent) {
+                throw new \BadMethodCallException('This method require a collection of Adherent entities');
```

Comment:
Also why not passing an ` AdherentCollection` instead to the method. This way, no need to check it contains `Adherent` instances.